index.bs

```diff
@@ -1719,6 +1719,56 @@ that should be considered in the development of new features, notably:
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3>Extend existing manifest files rather than creating new ones</h3>
+
+New web features should be self-contained and self-describing and ideally should not require an additional manifest file.
+Manifest files can add complexity to the development and deployment process for web technologies.  Sometimes manifest 
+files are required, but the use of different manifests on the web should be kept to a minimum.
+
+You may need to make a new manifest file if the domain of the manifest file is different from the existing manifest files.
+For example, if the fetch timing is different, or if the complexity of the manifest warrants it.  Application metadata should be added 
+to the Web App Manifest or be an extension of it. 
+Manifests designated to be used for specific applications or which require interoperability with non-browsers may need to take a different approach.
+Payment Method Manifest, Publication Manifest, and Origin Policy are examples of these cases.
```

Comment:
I don't think we should mention Publication Manifest, both here and below, as it uses JSON-LD, which we're at best undecided on.